src/main/java/com/nucleus/receipt/model/Receipt.java

```diff
@@ -2,37 +2,58 @@
 
 
 import com.nucleus.loanaplications.model.LoanApplications;
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.format.annotation.NumberFormat;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
 
 import javax.persistence.*;
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
 import java.time.LocalDate;
 
 @Entity
 @Table(name = "receipts")
 public class Receipt {
 
     @Id
+    @NotNull(message = "Receipt No. can not be null")
```

Comment:
for these validations (and the other ones), NotBlank might be better. otherwise its possible to add only a whitespace character and not have it throw errors